src/components/InfoBanner/blocks/InfoBannerCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { BoxProps, Box } from '../../Box/Box';
+import { Link as WaveLink } from '../../Link/Link';
+import { Text } from '../../Text/Text';
+
+import { ICON_VARIANTS, IconBox, InfoBannerVariants, ROLE_VARIANTS, RoundedBox } from '../InfoBanner';
+
+interface CardProps extends BoxProps {
+    children: ReactNode;
+    variant?: InfoBannerVariants;
+    emphasized?: boolean;
+}
+
+const LinkList = styled.ul`
```

Comment:
Maybe this can be `List` instead of `LinkList`? Does not seem to require its children to be links.

Reply:
The styling of this LinkList (padding) is actually quite coupled to the specs we are trying to meet (with links) with the addition of these compound components, as mentioned below in other response, I am not sure we would want the same for e.g. textual content in list items, that would naturally go into `InfoBannerCard.Description`.

Personally I would prefer to keep this naming to make that clear, that we don't want just anything to be put in there :)

Reply:
has been removed now 👍